### ISSUE TITLE
Please document how to pass env vars (e.g. secrets) to the build phase

### ISSUE THREAD

**Issue by holyjak** [into-docker/into-docker#22]
I need to download project dependencies from a private Maven repository, which requires authentication. Leiningen [supports this](https://github.com/technomancy/leiningen/blob/stable/doc/DEPLOY.md#authentication) e.g. via environment variables. However I see now way how to pass a custom env variable into it, when running `into build`. (Ideally, I would be able to do so also via the GitHub Action.)

**Comment by xsc** [comment]
@holyjak I was struggling a bit with finding a good way to allow custom environment variables, while making it harder to abuse the mechanism to create undocumented functionality in build scripts. Would be cool if you could check out #23 and give your opinion!

**Comment by xsc** [comment]
Released as [`1.1.0-RC3`](https://github.com/into-docker/into-docker/releases/tag/v1.1.0-RC3). You should be able to use it via the Github Action like so (sorry, untested as of now):

```yaml
- uses: into-docker/build-action@v1
  with:
    image: target-image:latest
    builder: intodocker/clojure
    version: 1.1.0-RC3
  env:
    LEIN_PASSWORD: ${{ secrets.LEIN_PASSWORD }}
```

**Comment by xsc** [comment]
Released in `1.1.0`.
